tests/testthat/test-funs-date.R

```diff
@@ -37,3 +37,24 @@ patrick::with_parameters_test_that(
   },
   fun = c("year", "month", "day", "quarter", "mday", "yday")
 )
+
+test_that("extracting date component", {
+  for_all(
+    tests = 20,
+    datetime = posixct_bounded(
+      left = as.POSIXct("2001-01-01 00:00:00", tz = "UTC"),
+      right = as.POSIXct("2015-01-01 00:00:00", tz = "UTC"),
+      any_na = TRUE
+    ),
+    property = function(datetime) {
+      test_df <- data.frame(x1 = ymd_hms(datetime, tz = "UTC"))
```

Comment:
```suggestion
      test_df <- data.frame(x1 = ymd_hms(datetime))
```

We want to test on datetimes with different timezones too, so this should be removed.